srcpkgs/QSyncthingTray/template

```diff
@@ -0,0 +1,39 @@
+# Template file for 'QSyncthingTray'
+pkgname=QSyncthingTray
+version=0.5.8
+revision=1
+build_style=cmake
+hostmakedepends="qt5-qmake qt5-host-tools"
+makedepends="qt5-declarative-devel qt5-webengine-devel qt5-webchannel-devel
+	qt5-location-devel qt5-quickcontrols2-devel qt5-devel"
+short_desc="Tray App for Syncthing written in C++"
+maintainer="Morgan Bazalgette <the@howl.moe>"
+license="LGPL-3.0-or-later"
+homepage="https://github.com/sieren/QSyncthingTray"
+distfiles="https://github.com/sieren/QSyncthingTray/archive/${version}.tar.gz"
+checksum=ff428ac5a09e5df7bde3d71044062de104a6cb17a2918ae317061f5ad1394480
+
+do_install() {
+	# Install main binary
+	vbin ./build/QSyncthingTray
+
+	# Desktop file
+	vmkdir usr/share/applications
+	vcopy ${FILESDIR}/qsyncthingtray.desktop usr/share/applications
+	# Download icons
+	cd ${FILESDIR}
```

Comment:
Nope. `$FILESDIR` does not get cleaned up automatically so you should not use it in the build process.
Just create a work folder for that inside `$wkrsrc`